hydra-node/test/Hydra/Chain/Direct/WalletSpec.hs

```diff
@@ -157,43 +155,39 @@ prop_balanceTransaction ::
 prop_balanceTransaction =
   forAllBlind (reasonablySized genValidatedTx) $ \tx ->
     forAllBlind (reasonablySized $ genOutputsForInputs tx) $ \lookupUtxo ->
-      forAllBlind (reasonablySized genUtxo) $ \walletUtxo ->
-        prop' lookupUtxo walletUtxo tx
- where
-  prop' lookupUtxo walletUtxo tx =
-    case coverFee_ pparams lookupUtxo walletUtxo tx of
-      Left{} ->
-        property True & label "Left"
-      Right (_, tx') ->
-        let inp' = knownInputBalance (lookupUtxo <> walletUtxo) tx'
-            out' = outputBalance tx'
-            out = outputBalance tx
-            fee = (txfee . body) tx'
-         in conjoin
-              [ coin (deltaValue out' inp') == fee
-              ]
-              & label "Right"
-              & counterexample ("Fee:             " <> show fee)
-              & counterexample ("Delta value:     " <> show (coin $ deltaValue out' inp'))
-              & counterexample ("Added value:     " <> show (coin inp'))
-              & counterexample ("Outputs after:   " <> show (coin out'))
-              & counterexample ("Outputs before:  " <> show (coin out))
+      forAllBlind genMarkedUtxo $ \walletUtxo ->
+        case coverFee_ pparams lookupUtxo walletUtxo tx of
+          Left err ->
+            property False & counterexample ("Error: " <> show err)
```

Comment:
Do we actually have 100% Right now :thinking:  ?

Reply:
We do :)